bridge-no-bootopts-net.ks.in

```diff
@@ -38,10 +38,17 @@ check_gui_configurations @KSTEST_NETDEV1@ bridge0
 check_device_connected bridge0 yes
 check_device_connected @KSTEST_NETDEV1@ yes
 
-# The bridge is created in initramfs so the slave connection is named @KSTEST_NETDEV1@
-# connection.master is identified by connection uuid so check __ANY
-check_device_config_value @KSTEST_NETDEV1@ BRIDGE bridge0 connection master __ANY
-check_device_config_value @KSTEST_NETDEV1@ BRIDGE bridge0 connection slave-type bridge
+if [ "@KSTEST_OS_NAME@" == "rhel" ] ; then
+    # RHEL-8-FAILURE: in rhel 8 he ifcfg is generated in intramfs by Anaconda
```

Comment:
the? although I think no article is needed there

Reply:
Fixed